README.md

```diff
@@ -16,30 +16,61 @@
   * [Join the Gitter chat room](https://gitter.im/cocotb)
 
 ## Installation
+Cocotb can be installed by running `pip install cocotb`, and requires:
 
-Cocotb can be installed by running `pip install cocotb`.
+- Python 3.5+
+- A C++ compiler
+- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/))
 
-## Quickstart
+For more details see [the documentation](https://cocotb.readthedocs.io/en/latest/quickstart.html#pre-requisites).
 
-    # Install pre-requisites (waveform viewer optional)
-    sudo yum install -y iverilog python-devel gtkwave
+## Usage
 
-    # Checkout git repositories
-    git clone https://github.com/cocotb/cocotb.git
+For a D flip-flop description:
 
-    # Install cocotb
-    pip install ./cocotb
+```verilog
```

Comment:
People often fail to understand what to do with those code snippets; add a comment at the beginning with a file name to give a hint:

```
# dff.sv
```